recipe/meta.yaml

```
@@ -1,39 +1,47 @@
 {% set name = "mkdocs-material-extensions" %}
-{% set version = "1.0.3" %}
+{% set version = "1.1" %}
```

```suggestion
{% set version = "1.1.1" %}
```
https://github.com/facelessuser/mkdocs-material-extensions/releases/tag/1.1.1

---

We can merge in this one (1.1) first, then the bot will pick up 1.1.1 next?

---

I don't see too much point in publishing a package which has already been superseded but if you prefer to do it that way, that's fine. As a maintainer you'll be able to merge version 1.1.1 yourself.